classes/Tools.php

```diff
@@ -2836,7 +2836,16 @@ public static function getRobotsContent()
             '*/modules/*.js',
             '*/modules/*.png',
             '*/modules/*.jpg',
```

Comment:
So, can you add .gif, woff, woff2 & ttf in modules too.

Reply:
Yes, I've changed this